app.js

```diff
@@ -0,0 +1,84 @@
+var path = require('path');
+var fs = require('fs');
+var express = require('express');
+var Router = require('react-router');
+
+var indexStaticWatcher = require('./lib/index-static-watcher').create();
+var PORT = process.env.PORT || 8008;
+var PRODUCTION = (process.env.NODE_ENV === 'production');
+var DIST_DIR = path.join(__dirname, 'dist');
+
+var indexStatic;
+var router;
+var app = express();
+
+var startProdApp = function() {
+  console.log([
+    'Production mode enabled. Note that "npm install" is assumed to',
+    'have recently been run with NODE_ENV="production". If this is not',
+    'the case, some or all static assets may be out of date.'
+  ].join('\n'));
+  indexStaticWatcher.build(function(err, newIndexStatic) {
+    if (err) {
+      throw err;
+    }
+
+    console.log('Built server-side bundle.');
```

Comment:
An aside: you might also be wondering what the heck the `1.1.bundle.js` file in `dist` is. That's actually mapbox! 

In #430 we used Webpack's code splitting functionality to asynchronously load mapbox on-demand only when the user is on the Clubs page. This is because we don't want to unnecessarily send the user all of mapbox if they e.g. only want to look at the homepage of the site. You can imagine how, as our site grows, we don't want to deliver a massive single JS bundle containing all the functionality the user could ever possibly use on our site--it's more efficient to only give the user what they need to view the page they requested, and then asynchronously load new chunks of code as needed.